core/src/main/java/org/bitcoinj/params/BitcoinNetworkParams.java

```diff
@@ -247,7 +247,11 @@ public MonetaryFormat getMonetaryFormat() {
         return new MonetaryFormat();
     }
 
+    /**
+     * @deprecated  use {@link ProtocolVersion#getBitcoinProtocolVersion()}
+     */
     @Override
+    @Deprecated
     public int getProtocolVersionNum(final ProtocolVersion version) {
         return version.getBitcoinProtocolVersion();
     }
```

Comment:
This deprecated method could be in `NetworkParameters`, and from its 2 subclasses removed.

A `deprecatedMembers()` test could be added. See 4e75444644c820f797bcef92f8cf0a66f59d4b1c as an example.

Reply:
Fixed and force-pushed.